src/error.rs

```diff
@@ -53,10 +53,8 @@ pub enum StellarSdkError {
 
     InvalidHexEncoding(FromHexError),
 
-    InvalidHashLength {
-        found_length: usize,
-        expected_length: usize,
-    },
```

Comment:
Is it safe to remove this?

Reply:
I mean if it is legacy code and not used anywhere, we can also remove it. 😅

Reply:
double checked again; there's none that i see it being used.

Reply:
Since we are probably the only ones using the sdk, it should be fine even if we use it elsewhere.